app/src/main/java/com/nervousfish/nervousfish/modules/pairing/AndroidBluetoothHandler.java

```diff
@@ -12,7 +12,8 @@
 import java.io.ObjectOutputStream;
 
 /**
- * Helper class for {@link AndroidBluetoothService}
+ * Helper class for {@link AndroidBluetoothService} and acts as the bridge between the client and
+ * the service.It simplifies the whole Bluetooth functionality to the methods start, connect, stop and send
```

Comment:
Because the client shouldn't access the service directly because that wouldn't be extensible. By using an adapter (actually, it's a bridge I think) we can also use non-Android Bluetooth services later on